actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -7,7 +7,7 @@ class Configuration
       attr_accessor :connection_class, :worker_pool_size
       attr_accessor :channel_load_paths
       attr_accessor :disable_request_forgery_protection, :allowed_request_origins
-      attr_accessor :cable, :url
```

Comment:
Good point - I'd say to have `action_cable_meta_tag` only talk with `mount_path`. `url` is no longer part of configuration with this PR.